etn-ccis/blui-react-component-library
# remove classes from combine function

Fixes https://github.com/etn-ccis/blui-react-component-library/issues/216.
Fixes https://github.com/etn-ccis/blui-react-component-library/issues/860.



#### Changes proposed in this Pull Request:

- remove classes.xxx on all combined classes
- rename defaultClasses to generatedClasses
- update components where combine not needed



#### Screenshots / Screen Recording (if applicable)

- New
![image](https://github.com/etn-ccis/blui-react-component-library/assets/119693939/21216a5c-cabf-4dbf-a83c-b25e96b05012)

- Old
![image](https://github.com/etn-ccis/blui-react-component-library/assets/119693939/0f598ddb-2fb4-417f-9481-98875aa02ff7)




#### To Test:

- git checkout bug/blui-5304-duplicated-classes
- yarn start:showcase
- open dev tools
- inspect alarms nav item
- verify BluiDrawerNavItem-title & BluiDrawerNavItem-nestedTitle are not duplicated.
- navigate to file demos/showcase/src/pages/contextual-page-templates/Dashboard.tsx
- after line 182 channelValue add ```classes={{ value: 'MyTestClass' }}``` and save
- inspect channelValue 
![image](https://github.com/etn-ccis/blui-react-component-library/assets/119693939/f7afbdf7-ba16-4226-82fb-99ed19eea1e5)
- verify MyTestClass is not duplicated
- test several other components and add custom classes



#### Any specific feedback you are looking for?

-
